spring-reactive-modules/spring-5-reactive/src/main/java/com/baeldung/websocket/ReactiveWebSocketConfiguration.java

```diff
@@ -3,26 +3,35 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import org.springframework.security.config.web.server.ServerHttpSecurity;
+import org.springframework.security.web.server.SecurityWebFilterChain;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.web.reactive.HandlerMapping;
 import org.springframework.web.reactive.handler.SimpleUrlHandlerMapping;
 import org.springframework.web.reactive.socket.WebSocketHandler;
+import org.springframework.web.reactive.socket.server.WebSocketService;
+import org.springframework.web.reactive.socket.server.support.HandshakeWebSocketService;
+import org.springframework.web.reactive.socket.server.upgrade.TomcatRequestUpgradeStrategy;
+import org.springframework.web.socket.config.annotation.EnableWebSocket;
 import org.springframework.web.reactive.socket.server.support.WebSocketHandlerAdapter;
+import org.springframework.web.socket.server.standard.ServerEndpointExporter;
+import org.springframework.context.annotation.Primary;
 
 @Configuration
+@EnableWebSocket
 public class ReactiveWebSocketConfiguration {
 
     @Autowired
     @Qualifier("ReactiveWebSocketHandler")
-    private WebSocketHandler webSocketHandler;
+    ReactiveWebSocketHandler reactiveWebSocketHandler;
```

Comment:
why you delete the `private` ?

Reply:
I have restored the privacy of this injected object.